openassessment/xblock/test/base.py

```diff
@@ -70,6 +70,17 @@
     'assess_type': 'full-grade'
 }
 
+TEAM_GOOD_ASSESSMENT_REGRADE = {
```

Comment:
Is there a reason for a new constant? Doesn't seem to follow the existing paradigm (only "regrade").

Reply:
I felt it would be a little cleaner to create a new one instead of deep copy.